### PR TITLE
Docs final touches

### PR DESCRIPTION
**Pull request by leina05**

### CODE REVIEW

File: new-docs/content/any/introduction.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Introduction
 any: true
 hideContents: true
+draft: true
```
Comment (saolsen) [inline new-docs/content/any/introduction.md:5]: Should this still be a draft?

File: new-docs/content/any/project/design-principles.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 ---
 title: Design Principles
-aliases: 
+aliases:
     - ../more/design-principles.html
+draft: true
```
Comment (saolsen) [inline new-docs/content/any/project/design-principles.md:5]: Should this still be a draft?

File: new-docs/content/any/project/internals.md [hunk 3]
```diff
@@ -40,3 +40,94 @@ library provides answers by inspecting application data, and control passes back
...
+
+
+## Data Filtering
+
+Oso supports applying authorization logic at the ORM layer so that you can
+efficiently authorize entire data sets. For example, suppose you have millions
+of posts in a social media application created by thousands of users, and
+regular users are only authorized to view posts from their friends. It would be
+inefficient to fetch all of the posts and authorize them one by one. It would
+be much more efficient to distill from the policy a _filter_ that can be
```
Comment (saolsen) [inline new-docs/content/any/project/internals.md:52]: maybe "wasteful and efficient" or "inefficient and better" instead of "inefficient and efficient"

**Review by saolsen — APPROVED** [review]

File: new-docs/content/any/guides/roles/rbac/index.md [hunk 4]
```diff
@@ -3,25 +3,24 @@ date: "2021-01-07T02:46:33.217Z"
...
+# Add Basic Roles to Your Application
 
-Many authorization systems in the wild are built on a [role-based access
-control](https://en.wikipedia.org/wiki/Role-based_access_control) model. The
-general thesis of RBAC is that the set of permissions for a system — a
-permission being the ability to perform an [action](glossary#actions) on a
-[resource](glossary#resources) — can be grouped into roles.
+Role-based access control (RBAC) groups permissions — a permission being the ability
+to perform an [action](glossary#actions) on a [resource](glossary#resources)
+— into roles. Roles are assigned to actors to confer these permissions.
```
[the remark this answers is not in the document]
Comment (leina05) [reply new-docs/content/any/guides/roles/rbac/index.md:17]: Yeah you right.

File: new-docs/content/any/introduction.md (hunk 1 above)
Comment (leina05) [reply new-docs/content/any/introduction.md:5]: yeah Graham 86ed it

File: new-docs/content/any/project/design-principles.md (hunk 2 above)
Comment (leina05) [reply new-docs/content/any/project/design-principles.md:5]: yes it's done

**Pull request merged**